src/reconcile/rec_write.c

```diff
@@ -48,8 +48,9 @@ typedef struct {
 	/* Track the page's maximum transaction ID. */
 	uint64_t max_txn;
 
-	uint64_t update_mem;		/* Total update memory */
-	uint64_t update_mem_skipped;	/* Skipped update memory */
+	uint64_t update_mem_all;	/* Total update memory size */
+	uint64_t update_mem_saved;	/* Saved update memory size */
+	uint64_t update_mem_uncommitted;/* Uncommitted update memory size */
```

Comment:
I'm inclined to leave these as-is even if they hold more information than we use at the moment. It feels like we are still iterating on the right solution, I've created:
https://jira.mongodb.org/browse/WT-3344
to make sure we don't forget to approach this more scientifically in the future.